packages/protocol/contracts/governance/LockedGold.sol

```diff
@@ -178,11 +178,14 @@ contract LockedGold is
     Balances storage account = balances[msg.sender];
     // Prevent unlocking gold when voting on governance proposals so that the gold cannot be
     // used to vote more than once.
-    require(!getGovernance().isVoting(msg.sender), "Account locked");
+    uint256 remainingLockedGold = getAccountTotalLockedGold(msg.sender).sub(value);
```

Comment:
why this variable is called `remaining`?

Reply:
remaining is meant as the gold that will remain in LockedGold contract once this transaction will go though. But naming is always the hardest part of programming :-) 

Would you have other suggestion how to name this variable ?